lib/active_storage_validations/matchers/size_validator_matcher.rb

```diff
@@ -49,6 +49,10 @@ def with_message(message)
         self
       end
 
+      def on(context)
+        @context = context
```

Comment:
@jozzi05 could you also initialise as `nil`?

Reply:
[733742f](https://github.com/igorkasyanchuk/active_storage_validations/pull/209/commits/733742fbc06ba0b4d7ee8feac08af051ff069121)